arcade/arcade/templates/{{ toolkit_name }}/pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "{{ package_name }}"
+version = "0.0.1"
+readme = "README.md"
+description = "{{ toolkit_description }}"
+authors = ["{{ toolkit_author_name }} <{{ toolkit_author_email }}>"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+arcade-ai = "{{ arcade_version }}"
+
+[tool.poetry.dev-dependencies]
+pytest = "^8.3.0"
+pytest-cov = "^4.0.0"
+mypy = "^1.5.1"
```

Comment:
should the extras for fastapi and evals for arcadeai be included here? `arcade dev` and `arcade evals` won't work without them. We should think about how we want to work with this long term too.

Reply:
I added `arcade-ai = {version = "{{ arcade_version }}", extras = ["fastapi", "evals"]}` to the toolkits dev dependencies. Lmk if this addresses your concern

Reply:
Reviewer note: The extras were later removed after a discussion between Eric and I. The dependency on `fastapi` is an _Arcade CLI_ dependency, it should not be a dependency declared by this toolkit.